published/010-2016-April-18.md

```diff
@@ -0,0 +1,67 @@
+
+### go-ipfs deps
+
+@whyrusleeping removed over 9000 (yes, over 9000) lines of unused godeps dependencies from the go-ipfs repo. Its a good deal more manageable now, and we are getting ever closer to having a purely gx managed package.
+
+### js-ipld
+
+![libp2p-cli](010_js-libp2p.png)
+
+This week @dignifiedquire shipped an updated version of [js-ipld](https://npmjs.org/ipld) which now conforms to the [latest spec of IPLD](https://github.com/ipfs/specs/tree/master/ipld). In addition [js-ipfs-ipld](https://npmjs.org/ipfs-ipld) was created, which implements the building blocks to use IPLD in js-ipfs. The third package that was published is [js-ipfs-cli](https://npmjs.org/ipld-cli) which gives you a cli tool to interact and experiment with IPLD. 
+
+#### `ipfs-blob-store` v1.0.0 released
+
+One of the benefits that comes from the new [Files API](link to best description we have) in 0.4.0, is that we can emulate any kind of file system expectations on top of IPFS, and by doing so, enable applications that use a file system, to use IPFS without any modification. One perfect example of that are all the applications that use a module that implements the [abstract-blob-store](https://github.com/maxogden/abstract-blob-store) interface, now with [ipfs-blob-store](https://github.com/ipfs/ipfs-blob-store), with a simple `npm install`, you can swap out your current blob-store implementation by the IPFS one.
+
+#### `js-ipfs-api` now supports the Files API
+
+On the same lines, `js-ipfs-api` now offers all the Files API calls through a convenient and easy to use library. You can find all of the function declarations on the [`js-ipfs-api` docs](https://github.com/ipfs/js-ipfs-api/blob/master/API.md#files)
+
+#### Making js-ipfs and js-libp2p modules awesome
+
+@JGAntunes, @dignifiedquire, @nginnever, @diasdavid, and @noffle have been working hard to bring [the pantheon of js-ipfs modules](https://github.com/ipfs/js-ipfs/issues?utf8=%E2%9C%93&q=is%3Aissue+coverage+dignified) up to snuff: detailed READMEs with examples and full API docs, as well as 100% test coverage. We made significant inroads last sprint on [js-peer-id](https://github.com/diasdavid/js-peer-id), [js-multiaddr](https://github.com/jbenet/js-multiaddr), [js-peer-info](https://github.com/diasdavid/js-peer-info), [js-ipfs-blocks](https://github.com/ipfs/js-ipfs-blocks/) and more! There are a lot more modules that still need good READMEs and docs, so [dive in](https://github.com/ipfs/js-ipfs/issues?utf8=%E2%9C%93&q=is%3Aissue+coverage+dignified+is%3Aopen) and give us a hand!
+
+## Community
+
+### Peergos
+
+Work on [Peergos](https://github.com/ianopolous/Peergos) is progressing well. We're close to getting a FUSE binding working. Throughput hasn't been as bad as we were expecting with all the encryption and erasure coding, and it's looking likely that it will end up being network IO bound, instead of CPU bound for non trivial writes. 
```

Comment:
change to third person, as this is ianopolous's project, not the core ipfs team's